src/i18n/locales/en.json

```diff
@@ -99,6 +104,6 @@
   "warning-https": "Most browsers do not allow video capture over an unencrypted connection (HTTP). Please switch to HTTPS.",
   "warning-recorder-not-supported": "Your browser does not support recording media streams.",
   "warning-recorder-safari-hint": "If you are using Safari, you can enable the experimental feature 'MediaRecorder' in settings. However, on macOS, we recommend switching to Chrome or Firefox.",
-  "warning-missing-connection-settings": "Connection to Opencast is not fully established: uploading is disabled. Please configure the connection in <1>the settings</1> (you won't lose your recording).",
+  "warning-missing-connection-settings": "Connection to Podcast server is not fully established: uploading is disabled. Please configure the connection in <1>the settings</1> (you won't lose your recording).",
```

Comment:
Reword and replace the link to settings with a "if the problem persists please contact <maillink>mediatechnologies@m.a.u</maillink>"

Reply:
this still needs to link to settings replaced with the directions how to report the problem to mediatech.